wps_scripts/src/main/groovy/org/noise_planet/noisemodelling/wps/Others_Tools/ZerodB_Source_From_Roads.groovy

```diff
@@ -0,0 +1,118 @@
+/**
+ * NoiseModelling is an open-source tool designed to produce environmental noise maps on very large urban areas. It can be used as a Java library or be controlled through a user friendly web interface.
+ *
+ * This version is developed by Université Gustave Eiffel and CNRS
+ * <http://noise-planet.org/noisemodelling.html>
+ *
+ * NoiseModelling is distributed under GPL 3 license. You can read a copy of this License in the file LICENCE provided with this software.
+ *
+ * Contact: contact@noise-planet.org
+ *
+ */
+/**
+ * @Author Valentin Le Bescond, Université Gustave Eiffel
+ */
+
+package org.noise_planet.noisemodelling.wps.Others_Tools
+
+import geoserver.GeoServer
+import geoserver.catalog.Store
+import groovy.sql.Sql
+import org.geotools.jdbc.JDBCDataStore
+import org.h2gis.utilities.wrapper.ConnectionWrapper
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+import java.sql.Connection
+
+title = 'Create 0db Source From Roads'
```

Comment:
Why this WPS block ? If this is for attenuation levels you must set -inf on each column

Reply:
It is indeed for attenuation levels. didn't know about the -inf thing. But i still need a script to go from a (Pk, THE_GEOM) table to a (PK, THE_GEOM, LWD63, LWD125, ...) table, don't I ? I can set it to -inf instead of 0 if you wish...